config.js

```diff
@@ -3,5 +3,16 @@ module.exports = {
     https: {
       key: fs.readFileSync('private_key.pem'),
       cert: fs.readFileSync('cert.pem')
+    },
+
+    db: {
+      type: 'mongo',
+      name: 'AnthroLinkDB',
+      host: 'localhost',
+      password: 'password',
+      port:     27017,
+      url:      '"mongodb://localhost:27017/AnthroLinkDB',
```

Comment:
Please remove url field and generate it using name,host,port  which is already present in the configuration.

Reply:
Keep Username and password next to each other. Strucuture looks little confusing. Username then password.